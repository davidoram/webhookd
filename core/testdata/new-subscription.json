{
    "name": "test-sub-1",
    "destination": {
        "kind": "webhook",
        "webhook": {
            "url": "https://localhost:9090/",
            "headers": [
                "X-API-KEY: e2315fb8-fe95-4d1d-a553-dd34ad1ea4db"
            ]
        }
    },
    "source": [
        {
            "topic": "topic-1",
            "jmes_filters": []
        }
    ],
    "configuration": {
        "batching": {
            "max_batch_size": 10,
            "max_batch_interval_seconds": 10
        }
    },
    "payload_size": {
        "max_payload_size_kb": 100
    },
    "retry": {
        "max_retries": 3,
        "retry_algorithm": "exponential_backoff"
    },
    "alerting": {
        "alert_channel": "email",
        "alert_emails": [
            "admin@client.com"
        ]
    }
}