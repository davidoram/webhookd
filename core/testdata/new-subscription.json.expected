{
    "name": "test-sub-1",
    "id": "7728eca9-c3d1-471b-aeb4-a6a9104e64b5",
    "source": {
        "kind": "topic-1"
    },
    "filter": {
        "jmes_filter": ""
    },
    "config": {
        "batch_size": 10,
        "max_wait": 10000000000
    },
    "created_at": "2024-01-08T01:09:28.732535Z",
    "updated_at": "2024-01-08T01:09:28.732535Z",
    "deleted_at": null,
    "destination": null
}